app/assets/javascripts/covid19/src/webapp/components/structures-table/cells/OrganismCell.tsx

```diff
@@ -13,7 +13,9 @@ export const OrganismCell: React.FC<CellProps> = React.memo(props => {
             .map(organism => ({
                 id: organism.id,
                 url: organism.externalLink,
-                tooltip: (
+                tooltip: !_.isEmpty(
+                    _.compact(_.values(_.pick(organism, ["id", "name", "commonName"])))
+                ) && (
                     <React.Fragment>
```

Comment:
I see that we are using a lot of this pattern of checking if fields exist and then render them. That's a bit verbose and we can forget fields.I was thinking of a _.compact of the react nodes? And also abstract the div of each node (it's very simple, yes, but used a lot of times)

Reply:
I removed the snippet from Ligands and Organisms. I was checking if fields were set on all cells tooltips because I've noticed that some entities had the tooltip void while entity.organism type was required but being, from my previous point of view, null on "runtime". So I doubtfully did that fix just in case in the others cells also have some value null.
After this comment, I checked if the other cells could have a null value being shown as entity.organism but I noticed it couldn't. In Covid19InfoFromJsonRepository:198 we are already checking if the organism is null so it was just a misleading type on Covid19Data.types. Now this snippet only applies to OrganismCell, so I followed your comment there.